twilio_test.rb

```diff
@@ -0,0 +1,12 @@
+require 'rubygems'
+require 'twilio-ruby'
+require_relative '../.env.rb'
+# Get your Account Sid and Auth Token from twilio.com/user/account
+ENV[:account_sid]
+ENV[:auth_token]
+@client = Twilio::REST::Client.new account_sid, auth_token
+
+message = @client.account.messages.create(:body => "Jenny please?! I love you <3",
```

Comment:
Metrics/LineLength: Line is too long. [82/80]<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax.